osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacySwell.cs

```diff
@@ -0,0 +1,136 @@
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Framework.Audio.Sample;
+using osu.Game.Audio;
+using osuTK;
+using osu.Game.Rulesets.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Taiko.Skinning.Legacy
+{
+    public partial class LegacySwell : Container, ISkinnableSwell
+    {
+        private Container bodyContainer = null!;
+        private Sprite spinnerCircle = null!;
+        private Sprite shrinkingRing = null!;
+        private Sprite clearAnimation = null!;
+        private ISample? clearSample;
+        private LegacySpriteText remainingHitsCountdown = null!;
+
+        private bool samplePlayed;
+
+        public LegacySwell()
+        {
+            Anchor = Anchor.Centre;
+            Origin = Anchor.Centre;
+            RelativeSizeAxes = Axes.Both;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(ISkinSource skin, SkinManager skinManager)
+        {
+            Child = new Container
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Position = new Vector2(200f, 100f),
+
+                Children = new Drawable[]
+                {
+                    bodyContainer = new Container
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Alpha = 0,
+
+                        Children = new Drawable[]
+                        {
+                            spinnerCircle = new Sprite
+                            {
+                                Texture = skin.GetTexture("spinner-circle"),
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                Scale = new Vector2(0.8f),
+                            },
+                            shrinkingRing = new Sprite
+                            {
+                                Texture = skin.GetTexture("spinner-approachcircle") ?? skinManager.DefaultClassicSkin.GetTexture("spinner-approachcircle"),
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                Scale = Vector2.One,
+                            },
+                            remainingHitsCountdown = new LegacySpriteText(LegacyFont.Combo)
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                Position = new Vector2(0f, 165f),
+                                Scale = Vector2.One,
+                            },
+                        }
+                    },
+                    clearAnimation = new Sprite
+                    {
+                        // File extension is included here because of a GetTexture limitation, see #21543
+                        Texture = skin.GetTexture("spinner-osu.png"),
```

Comment:
This workaround can't exist. Whatever is causing the lookup failure of the texture needs to be fixed at another level entirely in a separate pull request.